Packs/AnomaliThreatStreamFeed/Integrations/AnomaliThreatStreamFeed/AnomaliThreatStreamFeed.yml

```diff
@@ -0,0 +1,158 @@
+category: Data Enrichment & Threat Intelligence
+sectionOrder:
+- Connect
+- Collect
+commonfields:
+  id: Anomali ThreatStream Feed
+  version: -1
+configuration:
+- display: Fetch indicators
+  defaultvalue: 'true'
+  name: feed
+  type: 8
+  section: Collect
+  additionalinfo: Enable this checkbox to automatically pull indicators from the Anomali ThreatStream at regular intervals.
+- display: Fetch by
+  name: fetchBy
+  type: 15
+  defaultvalue: "Created Time"
```

Comment:
@Shir2611 - Please verify with @dmitryzinger and @meital-aizen what should be the default here. 
According to @Ni-Knight's last message, it should be modified (like the original design doc).